### ISSUE TITLE
[api-extractor] Error: The expression contains an import() type, which is not yet supported by API Extractor

### ISSUE THREAD

**Issue by imtaotao** [microsoft/rushstack#2140]
## Please prefix the issue title with the project name i.e. [rush], [api-extractor] etc. ##

**Is this a feature or a bug?**

- [ ] Feature
- [x] Bug

**Please describe the actual behavior.**
This error occurred when I merged types.

**index.ts**
```ts
import { channel } from './channel';
const Gar = { channel };
```
**index.d.ts**
```ts
/// <reference types="node" />
import { channel } from './channel';
declare const Gar: {
    channel: import("events").EventEmitter;
};
```

**./channel.ts**
```js
import { EventEmitter } from 'events';
export const channel = new EventEmitter();
```

**./channel.d.ts**
```ts
/// <reference types="node" />
import { EventEmitter } from 'events';
export declare const channel: EventEmitter;
```

**What is the expected behavior?**
The above `.d.ts` is generated by tsc. I hope this syntax is allowed，no error happend  😂  , Thank you.

**If this is a bug, please provide the tool version, Node.js version, and OS.**

* **Tool:** 
* **Tool Version:*7.9.2* 
* **Node Version:*v12.18.2* 
  * **Is this a LTS version?*no* 
  * **Have you tested on a LTS version?*no* 
* **OS:**

**Comment by imtaotao** [comment]
I'm have set `api-extractor.json`, but still can't found resovle solution 😭.

child json
```json
{
  "extends": "../../api-extractor.json",
  "mainEntryPointFilePath": "./dist/packages/<unscopedPackageName>/src/index.d.ts",
  "dtsRollup": {
    "publicTrimmedFilePath": "./dist/<unscopedPackageName>.d.ts"
  }
}
```

parent json
```json
{
  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",

  "apiReport": {
    "enabled": true,
    "reportFolder": "<projectFolder>/temp/"
  },

  "docModel": {
    "enabled": true
  },

  "dtsRollup": {
    "enabled": true
  },

  "tsdocMetadata": {
    "enabled": false
  },

  "messages": {
    "compilerMessageReporting": {
      "default": {
        "logLevel": "warning"
      }
    },

    "extractorMessageReporting": {
      "default": {
        "logLevel": "warning",
        "addToApiReportFile": true
      },

      "ae-missing-release-tag": {
        "logLevel": "none"
      }
    },

    "tsdocMessageReporting": {
      "default": {
        "logLevel": "warning"
      },

      "tsdoc-undefined-tag": {
        "logLevel": "none"
      }
    }
  }
}

```

**Comment by achrinza** [comment]
This is a known limitation: #1050

Try adding `import { EventEmitter } from 'events';` to `index.ts` to prevent [lazy imports](https://www.typescriptlang.org/docs/handbook/modules.html#optional-module-loading-and-other-advanced-loading-scenarios).

**Comment by imtaotao** [comment]
Add `import { EventEmitter } from 'events'` can indeed solve this problem, but actually, not useing `EventEmitter`  may make my colleague delete this line of code

**Comment by kasperisager** [comment]
I imagine this has been solved by #1916 as well.

**Comment by octogonz** [comment]
> 
> I imagine this has been solved by #1916 as well.

It has! Check out API Extractor 7.18.0! 🎈
